### PR TITLE
[ElasticSearch Curator]: adding the helm-stable upstream elasticsearch-curator chart

### PR DESCRIPTION
**Pull request by alejandroEsc**
Including the (soon to be deprecated) stable helm elasticsearch-curator chart to this repository. 

- [X] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [X] README.md updated with any new values or changes
- [ ] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

File: elasticsearch-curator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: "5.7.6"
+description: DEPRECATED A Helm chart for Elasticsearch Curator
```
Comment (joejulian) [inline elasticsearch-curator/Chart.yaml:3]:
```suggestion
description: A Helm chart for Elasticsearch Curator
```

File: elasticsearch-curator/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+version: 2.2.3
+home: https://github.com/elastic/curator
+keywords:
+- curator
+- elasticsearch
+- elasticsearch-curator
+sources:
+- https://github.com/kubernetes/charts/elasticsearch-curator
+- https://github.com/pires/docker-elasticsearch-curator
+deprecated: true
```
Comment (joejulian) [inline elasticsearch-curator/Chart.yaml:14]:
```suggestion
```

**Review by joejulian — COMMENTED** [review]
Should we un-deprecate this as part of this PR?

**Comment by alejandroEsc** [comment]
> Should we un-deprecate this as part of this PR?

good catches, thank you!

**Review by joejulian — APPROVED** [review]

**Comment by jmlrt** [comment]
Hey @alejandroEsc,

Thanks for your work on this PR. However, this is not something that we'll want to integrate in Elastic official charts. 

Indeed, Elastic now recommends using [ILM](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html) and [SLM](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-lifecycle-management.html) instead of Curator (see https://github.com/elastic/helm-charts/issues/236). Also, Elastic doesn't provide any official Curator Docker image for the reasons above and we stick to use only Docker images from Elastic in this charts.

**Comment by joejulian** [comment]
Am I reading this correctly? You're refusing to foster community and, instead, requiring paid subscriptions to have the basic functionality that curator provides?

**Comment by alejandroEsc** [comment]
I am sorry to hear you guys are choosing to go on this route, I'll be looking elsewhere. Thank you.

**Comment by jmlrt** [comment]
>You're refusing to foster community and, instead, requiring paid subscriptions to have the basic functionality that curator provides?

@joejulian, FYI ILM and SLM are available in Elasticsearch Basic free version as per https://www.elastic.co/subscriptions.

Meanwhile, you can still use Curator if it fits your needs and are free to create a Docker image and Helm chart for it and host them.

**Comment by guyelia** [comment]
@alejandroEsc have you found any alternative?

we in the same situation.

**Pull request closed without merging**